make/kube

```diff
@@ -25,5 +25,9 @@ rm -rf "${FISSILE_OUTPUT_DIR}"
 
 fissile build "${BUILD_TARGET}"
 
+if [ "${BUILD_TARGET}" = "helm" ]; then
+    cp Chart.yaml ./helm
```

Comment:
Actually, the copy target here should be `${FISSILE_OUTPUT_DIR}` (it's what you deleted) right?

Reply:
Yes, and uaa needs the fix as well.